README.md

```diff
@@ -46,21 +46,21 @@
 
 # Features
 
-| **User Interface**    |  **Secure Storage**  |**Interoperability**   |  **Standards & Protocols** | **Connections & Credential Issuance**  |     **SSI Agents**   |           
-|----------------|---------------|---------------|----------------|---------------|----------------|
-|  Intuitive User Experience |  Hardware Security Module (HSM) or Secure Enclave (SE) securely stores private key material |  [Aries RFCs](https://github.com/hyperledger/aries-rfcs) |  [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) |  Holder-to-Holder |  [Open Wallet Foundation](https://github.com/openwallet-foundation/agent-framework-javascript)| 
-|  Android and iOS support |  Hierarchical Deterministic (HD) Identity Wallet | [W3C](https://www.w3.org/) | [Autonomic Identifier](https://weboftrust.github.io/WOT-terms/docs/glossary/autonomic-identifier) (AID)  |  Issuer-to-Holder |  [KERIA](https://github.com/cardano-foundation/keria) | 
-|  | | [DIDComm](https://github.com/decentralized-identity/didcomm-messaging) |  [ACDC](https://wiki.trustoverip.org/display/HOME/ACDC+%28Authentic+Chained+Data+Container%29+Task+Force) |  [Credential Issuance Testing Tool](https://identity-wallet-credential-issuance-web-interface.vercel.app/)  |  [Signify-TS](https://github.com/cardano-foundation/signify-ts) | 
-|  | | [KERI on Cardano](https://github.com/cardano-foundation/cardano-backer) | |  | |
+| **User Interface**    |  **Secure Storage**  |  **Standards & Protocols** |  **SSI Services**   |           
+|----------------|---------------|---------------|----------------|
+|  Intuitive User Experience |  Hardware Security Module (HSM)   | [KERI on Cardano](https://github.com/cardano-foundation/cardano-backer) | [KERIA](https://github.com/cardano-foundation/keria) | 
+|  Android  |  Secure Enclave (SE) | [Autonomic Identifier](https://weboftrust.github.io/WOT-terms/docs/glossary/autonomic-identifier) |  [Signify-TS](https://github.com/cardano-foundation/signify-ts) | 
```

Comment:
Standards for KERI at at https://github.com/WebOfTrust/keri - I would say the most prominent are KERI, CESR and ACDC